doc/release/technotes/README.llvm.rst

```diff
@@ -51,63 +56,65 @@ Note:
 Activating the LLVM support
 ---------------------------
 
-To compile a program using the LLVM backend, add --llvm to the chpl command
+To compile a program using the LLVM backend, add ``--llvm`` to the chpl command
 line.
 
-If you pass a --savec directory, the LLVM backend will emit two .bc files
+If you pass a ``--savec`` directory, the LLVM backend will emit two .bc files
 in that directory:
+
 * chpl__module.bc is the version that will be linked
 * chpl__module-nopt.bc is the generated code without optimizations applied.
 
-Passing --fast will cause LLVM optimizations to run.
+Passing ``--fast`` will cause LLVM optimizations to run.
 
-The --ccflags option can control which LLVM optimizations are run, using the
+The ``--ccflags`` option can control which LLVM optimizations are run, using the
 same syntax as flags to clang.
 
```

Comment:
I don't know what you mean. Do you want clang to be in `` ? the --ccflags semes OK...